### ISSUE TITLE
[rcamera] Matrix definition missing in rcamera.h 

### ISSUE THREAD

**Issue by 9tin9tin9** [raysan5/raylib#3160]
**WARNING: Please, read this note carefully before submitting a new issue:**

It is important to realise that **this is NOT A SUPPORT FORUM**, this is for reproducible BUGS with raylib ONLY.

There are lots of generous and helpful people ready to help you out on [raylib Discord forum](https://discord.gg/raylib) or [raylib reddit](https://www.reddit.com/r/raylib/).

Remember that asking for support questions here actively takes developer time away from improving raylib.

---

Please, before submitting a new issue verify and check:

 - [x] I tested it on latest raylib version from master branch
 - [x] I checked there is no similar issue already reported
 - [x] My code has no errors or misuse of raylib

### Issue description

These two functions uses `Matrix` but it is not defined in rcamera.h
```C
RLAPI Matrix GetCameraViewMatrix(Camera *camera);
RLAPI Matrix GetCameraProjectionMatrix(Camera* camera, float aspect);
```

I see `Vector2`, `Vector3`, `Camera3D` etc defined but not `Matrix` in `#if defined(CAMERA_STANDALONE)` section.
Missing `Matrix` definition makes rcamera.h cannot be used as standalone module.

**Comment by raysan5** [comment]
@9tin9tin9 Good catch! `rcamera.h` was rewriten from scratch recently and it was probably removed by error.
